runtime/lua/vim/_meta.lua

```diff
@@ -198,7 +198,10 @@ end
 --                  Can be done in a separate PR.
 local key_value_options = {
   fillchars = true,
+  fcs = true,
   listchars = true,
+  lcs = true,
+  winhighlight = true,
   winhl = true,
```

Comment:
Looks like the long vs. short option wasn't used consistently here, so this is good. 

How about the rest of this API? Does it cover both full name and abbreviation?

Reply:
Thanks, that makes sense.

And agreed, that should be reflected in the metadata (via `list='dict'` or somesuch), but that is something for 0.9.